dev/src/cli/cli_create.ts

```diff
@@ -0,0 +1,262 @@
+  }
+  await saveToFile(dotEnvPath, lines.join('\n'));
+}
+
+export async function createAgent(options: AgentCreationOptions) {
+  const agentDir = path.join(dirname, options.agentName);
+  await generateAgentFolder(agentDir);
+
+  if (!options.model) {
+    const model: symbol | string = (await select(
+      {
+        message: 'Choose a model for the root agent',
+        options: [
+          { label: 'gemini-2.5-flash', value: 'gemini-2.5-flash' },
+          { label: 'gemini-2.5-pro', value: 'gemini-2.5-pro' },
+          { label: 'gemini-3-pro-preview', value: 'gemini-3-pro-preview' },
+          { label: 'gemini-3-flash-preview', value: 'gemini-3-flash-preview' },
+        ],
+      },
+    ));
+    if(isCancel(model)) {
+      process.exit(0);
+    }
+    options.model = model;
+  }
+  
+
+  if(options.language !== "js" && options.language !== "ts") {
+    const language = (await select({
+        message: 'Choose a language for the agent',
+        options: [
+          { label: 'JavaScript', value: 'js' },
+          { label: 'TypeScript', value: 'ts' },
+        ],
+      },
+    ));
+    if(isCancel(language)) {
+      process.exit(0);
+    }
+    options.language = language;
+  }
+
+  if (!options.apiKey && !options.project) {
+    const backend: symbol | string = (await select({
+        message: 'Choose a backend',
+        options: [
+          { label: 'Google AI', value: 'googleai' },
+          { label: 'Vertex AI', value: 'vertex' },
+        ],
+      },
+    ));
+    if(isCancel(backend)) {
+      process.exit(0);
+    }
+
+    if (backend === 'vertex') {
+      const projectResponse: string = (await text(
+        {
+          message: 'Enter the Google Cloud Project ID',
+          initialValue: await getGcpProject(),
+        },
+      )).toString();
+      if(isCancel(projectResponse)) {
+        process.exit(0);
+      }
+      options.project = projectResponse;
+
+      const regionResponse: symbol | string = (await text({
+          message: 'Enter the Google Cloud Region',
+          initialValue: await getGcpRegion(),
+        },
+      ));
+      if(isCancel(regionResponse)) {
+        process.exit(0);
+      }
+      options.region = regionResponse;
+    } else {
+      const apiKeyResponse: symbol | string = (await text(
+        {
+          message: 'Enter the Google API Key',
+        },
+      ));
+      if(isCancel(apiKeyResponse)) {
+        process.exit(0);
+      }
+      options.apiKey = apiKeyResponse;
+    }
+  }
+
+  await generateFiles(options);
+  await saveToFile(path.join(agentDir, 'package.json'), PACKAGE_JSON(options.agentName, options.language));
+  if (options.language === "ts") {
+    await await execPromise(`npm install typescript --save-dev`, { cwd: options.agentName });
+    await saveToFile(path.join(agentDir, 'tsconfig.json'), TS_CONFIG);
+  }
+  await execPromise(`npm install @google/adk @google/adk-devtools zod@3.25.76`, { cwd: options.agentName });
```

Comment:
not sure if we need the `@google/adk-devtools` here.

Because if user generates the new project then they already using the `@google/adk-devtools`

Reply:
You're right. Removed